modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -74,18 +76,87 @@ Currently, not all tests run selectively on the nodes belonging to the pool.
 
 [id="cnf-performing-end-to-end-tests-running-the-tests_{context}"]
 == Running the tests
-Assuming the file is in the current folder, the command for running the test suite is:
+Assuming the kubeconfig file is in the current folder, the command for running the test suite is:
 
 ----
 $ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig registry.redhat.io/openshift4/cnf-tests-rhel8:v4.6 /usr/bin/test-run.sh
 ----
 
 This allows your kubeconfig file to be consumed from inside the running container.
 
+[id="cnf-performing-end-to-end-tests-running-the-latency_tests_{context}"]
+=== Running the latency tests
+In OpenShift Container Platform 4.7, you can also run latency tests from the CNF-test container. The latency test allows you to set a latency limit, so that you can determine performance, throughput, and latency.
+
+The latency test runs the `oslat` tool, which is an open source program to detect OS level latency. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/5315541[How to measure OS and hardware latency on isolated CPUs?].
+
+By default, the latency tests are disabled. To enable the latency test, you must add the `LATENCY_TEST_RUN` variable and set its value to true. For example, `LATENCY_TEST_RUN=true`.
+
+Additionally, you can set the following environment variables for latency tests:
+
+* `LATENCY_TEST_RUNTIME` - Specifies the amount of time (in seconds) that the latency test must run.
+* `OSLAT_MAXIMUM_LATENCY` - Specifies the maximum latency (in microseconds) that is expected from all buckets during the `oslat` test run.
+
+To perform the latency tests, run the following command:
+
+----
+docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e LATENCY_TEST_RUN=true -e LATENCY_TEST_RUNTIME=600 -e OSLAT_MAXIMUM_LATENCY=20 registry.redhat.io/openshift4/cnf-tests-rhel8:v4.7 /usr/bin/test-run.sh
+----
+[NOTE]
+====
+You must run the latency test in link:cnf-performing-end-to-end-tests-disconnected-mode_{context}[Discovery mode].
+====
+
+Here is an excerpt of a sample result of a 10 second latency test:
```

Comment:
Instead of a "Here is" statement, you can just use this formatting to label the example:


    .Excerpt of a ample result of a 10 second latency test